Sources/Classes/RSClient.m

```diff
@@ -414,6 +425,10 @@ + (RSOption*) getDefaultOptions {
     return _defaultOptions;
 }
 
+- (RSOption *)defaultOptions {
+    return _instance.defaultOptions;
+}
```

Comment:
This is entering into an infinite loop.

Steps to reproduce it: Just enable lifecycle and run the app.